File: Heat/ViewModels/Invoices/InvoiceRowViewModel.vb
Imports System.ComponentModel.DataAnnotations


Namespace ViewModels.Invoices
    Public Class InvoiceRowViewModel

        <Key> _
        Property ID As Integer

        Property InvoiceID As Integer

        <Display(name:="Item")> _
        Property Item As Integer

        <Display(name:="Prodotto")> _
        Property Product As String

        <Display(name:="Quantità")> _
        Property Quantity As Single

        <Display(name:="Prezzo")> _
        Property UnitPrice As Decimal

        <Display(name:="IVA")> _
        Property VAT As Single

        <Display(name:="Sconto 1")> _
        Property Discount1 As Single

        <Display(name:="Sconto 2")> _
        Property Discount2 As Single

        <Display(name:="Sconto 3")> _
        Property Discount3 As Single

        <Display(name:="Totale imponibile")> _
        Property TotalBeforeTax As Decimal

        <Display(name:="TOTALE")> _
        Property Total As Decimal


    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/InvoiceRowViewModel.cs
using System.ComponentModel.DataAnnotations;


namespace Heat.ViewModels.Invoices
{
    public class InvoiceRowViewModel
	{

		[Key()]
		public int ID { get; set; }

		public int InvoiceID { get; set; }

		[Display(Name = "Item")]
		public int Item { get; set; }

		[Display(Name = "Prodotto")]
		public string Product { get; set; }

		[Display(Name = "Quantità")]
		public float Quantity { get; set; }

		[Display(Name = "Prezzo")]
		public decimal UnitPrice { get; set; }

		[Display(Name = "IVA")]
		public float VAT { get; set; }

		[Display(Name = "Sconto 1")]
		public float Discount1 { get; set; }

		[Display(Name = "Sconto 2")]
		public float Discount2 { get; set; }

		[Display(Name = "Sconto 3")]
		public float Discount3 { get; set; }

		[Display(Name = "Totale imponibile")]
		public decimal TotalBeforeTax { get; set; }

		[Display(Name = "TOTALE")]
		public decimal Total { get; set; }


	}

}
